### DISCUSSION TITLE
SSH LocalForward?

### DISCUSSION THREAD

**Discussion by rawhat** [wezterm/wezterm#1272] [Q&A]
Hi!  First off, super huge thanks @wez for making this.  Primarily developing on WSL has presented some... challenges finding a satisfactory terminal.  Fortunately this exists, and I have been thoroughly enjoying it.

I recently got a laptop for work, and so have been trying to shift to developing on that from my home desktop.  For ease / comfort, I was using tmux since I am familiar.  However, I got very used to the wezterm tab workflow, and I would much much prefer to use that.

I looked into the ssh and multiplexing setup and tried both of them.  They seem super great, and I was very excited.  Unfortunately, they don't seem to support local forwarding?  I'm not sure if that's some misconfiguration on my part, or a lack of libssl2 support, or just that it's a relatively new feature.  I have tried passing `-oLocalForward=...` but it doesn't seem to be accessible.  I don't think it's a hosts file issue on Windows, as I am able to connect via plain SSH in a terminal.

Is this something feasible or on the radar?  I have a few workarounds (tmux, as mentioned, or just ssh-ing in two tabs), but obviously a native wezterm solution would be awesome.

Thanks!

For reference, this is what I am running:
```& 'C:\Program Files\WezTerm\wezterm-gui.exe' ssh -oLocalForward=5555:localhost:5555 -oLocalForward=4001:localhost:4001 -oLocalForward=8081:localhost:8080 alex@xxx.xxx.xxx.xxx```

**Comment by wez** [comment]
Sorry for the delay in responding; I've recently moved to a new home in a new state so I've not had a lot of time to keep on top of wezterm over the past few weeks.

wezterm doesn't currently support ssh port forwarding for its built-in ssh client or mux.

I've filed https://github.com/wez/wezterm/issues/1341 to track adding it

**Reply by rawhat** [reply]
No worries at all.  Hope the new place is working out for you!

Thanks for making the issue, I will keep an eye on it.  I appreciate all the hard work you have put into this.

Best of luck in the new location!
